### ISSUE TITLE
Disk Usage Cache - Constantly Purged

### ISSUE THREAD

**Issue by tibbie123** [engintron/engintron#1197]
The cache is being constantly purged. The furthest that is ever cached are categories - the highest dynamic has ever got is 80M. My sites have 15G mainly product. The sites are not super busy so changed params_dynamic   proxy_cache_valid              200 7d; ( have tried all sorts of times like m and h) and params_static  proxy_cache_valid       200 301 302 7d

I have made NO other adjustments. Please advise what I am missing. Thanks

---------------------------------------
Nginx Cache/Temp Disk Usage:
52M	/var/cache/nginx/engintron_dynamic
23M	/var/cache/nginx/engintron_static
4.0K	/var/cache/nginx/engintron_temp System Time: Sat May 30 06:44:15 EDT 2020
--------------------------------------
Nginx Cache/Temp Disk Usage:
50M	/var/cache/nginx/engintron_dynamic
23M	/var/cache/nginx/engintron_static
4.0K	/var/cache/nginx/engintron_temp

System Time: Sat May 30 06:49:59 EDT 2020
---------------------------------------

**Comment by fevangelou** [comment]
You also need to bump the limits in /etc/nginx/nginx.conf. Update these 2 lines https://github.com/engintron/engintron/blob/42713cd3d5f446d20536a43e6c00ee6455c07534/nginx/nginx.conf#L147-L148 with:
```
    proxy_cache_path  /var/cache/nginx/engintron_dynamic  levels=1:2 keys_zone=engintron_dynamic:512m max_size=4g inactive=12h use_temp_path=off;
    proxy_cache_path  /var/cache/nginx/engintron_static   levels=1:2 keys_zone=engintron_static:512m  max_size=4g inactive=12h use_temp_path=off;
```

Keep in mind that unless all your sites are WordPress or full into the exclusion rules or don't have user-generated content (e.g. they are NOT eshops, forums etc.), you should NOT use a 7 day cache. Realistically speaking, most of the times you won't need more than a few hours for ultra aggressive caching.

**Comment by tibbie123** [comment]
Thanks for the fast reply! I have copied and pasted as suggested - I just changed the keys_zone=engintron to 3072 dynamic and static. 
All my sites are Wordpress and there is no user generated content anyway. Have hugely reduced the cache from the 7 days as suggested.  
Am currently watching the server load going done to about the lowest I have ever seen. Also available memory going up.
Thanks again!
